plugins/base/base-test-utils/src/main/kotlin/renderers/defaultSourceSet.kt

```diff
@@ -1,30 +1,3 @@
 package renderers
 
-import org.jetbrains.dokka.DokkaSourceSetID
-import org.jetbrains.dokka.DokkaSourceSetImpl
-import org.jetbrains.dokka.Platform
-
-val defaultSourceSet = DokkaSourceSetImpl(
-    moduleDisplayName = "DEFAULT",
-    displayName = "DEFAULT",
-    sourceSetID = DokkaSourceSetID("DEFAULT", "DEFAULT"),
-    classpath = emptyList(),
-    sourceRoots = emptySet(),
-    dependentSourceSets = emptySet(),
-    samples = emptySet(),
-    includes = emptySet(),
-    includeNonPublic = false,
-    reportUndocumented = false,
-    skipEmptyPackages = true,
-    skipDeprecated = false,
-    jdkVersion = 8,
-    sourceLinks = emptySet(),
-    perPackageOptions = emptyList(),
-    externalDocumentationLinks = emptySet(),
-    languageVersion = null,
-    apiVersion = null,
-    noStdlibLink = false,
-    noJdkLink = false,
-    suppressedFiles = emptySet(),
-    analysisPlatform = Platform.DEFAULT
-)
+val defaultSourceSet = testApi.testRunner.defaultSourceSet
```

Comment:
Do we really need this file?

Reply:
We would not, but that would require me to change a lot usages to refer to the new path. I putted this into a "lower" module, since I needed it there 🙄  
Do you want me to go ahead changing all usages, or should I go with a Deprecated annotation?

Reply:
Annotating it as deprecated would produce lots of warnings, wouldn't it?
I know that removing it will be a tedious process but for the sake of project we shouldn't have leftovers such as this in our repo. 😒

Reply:
Done ✅ ☺️